release-notes/unreleased/2263.feat.rst

```diff
@@ -0,0 +1,7 @@
+The ``private`` option under :class:`EnvironmentOptions` is now supported on the 
+``ibm_cloud`` channel (new IBM Quantum Platform). When this option
```

Comment:
Once the new channel is introduced, this reno should also talk about "ibm_quantum_platform"